pandas/io/formats/format.py

```diff
@@ -103,14 +106,14 @@
         * inherit
         * match-parent
         * initial
-        * unset
+        * unset.
 """
 
 return_docstring = """
-
     Returns
```

Comment:
The Returns section should go after the Parameters section and before the See Also section.